bases/renku_data_services/data_tasks/config.py

```diff
@@ -20,12 +20,14 @@ class PosthogConfig:
     environment: str
```

Comment:
I know it is like that from before and you did not touch it here. But lets make the api_key have `field(repr=False)`